.goreleaser.yml

```diff
@@ -120,3 +123,25 @@ docker_manifests:
   - name_template: "{{.Env.IMAGE_REGISTRY}}/{{.Env.MIGRATOR_IMAGE_REPOSITORY}}:{{ .Env.IMAGE_TAG }}"
     image_templates:
       - "{{.Env.IMAGE_REGISTRY}}/{{.Env.MIGRATOR_IMAGE_REPOSITORY}}:{{ .Env.IMAGE_TAG }}"
+
+archives:
+  - id: botkube-cli
+    builds:
+      - botkube-cli
+brews:
+  - name: botkube
+    ids:
+      - botkube-cli
+    install:
+      bin.install "botkube"
+    tap:
+      owner: "{{.Env.HOMEBREW_REPO_OWNER}}"
+      name: "{{.Env.HOMEBREW_REPO_NAME}}"
+      token: "{{ .Env.GITHUB_TOKEN }}"
+    download_strategy: CurlDownloadStrategy
+    commit_msg_template: "Brew formula update for {{ .ProjectName }} version {{ .Tag }}"
+    folder: Formula
+    homepage: "https://botkube.io"
+    description: "BotKube CLI is a command line tool that helps you migrate your Botkube installation to Botkube Cloud."
```

Comment:
I think the long-term plan is `botkube` will be a generic CLI. I'd use some more generic phrase here too, so that we don't have to change it when we add more features.

Reply:
Any suggestions? For example, `Botkube CLI is a command line tool for managing both open source and cloud instances`?

Reply:
`Botkube CLI is a command line tool for managing Botkube instances` - most basic  brew repo description